packages/web-components/src/global/main.css

```diff
@@ -9,3 +9,7 @@ va-process-list li :is(h2, h3, h4, h5) {
   clear: none;
   padding-top: 0.188rem;
 }
+
+va-promo-banner {
+  outline: none !important;
```

Comment:
Can you do this without the `!important` keyword (make it more specific)? For example:

```css
va-promo-banner[href]:focus {
  outline: none;
}
```

Reply:
Actually, it doesn't work without the `!important` keyword.